recipes/pymongoarrow/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "pymongoarrow" %}
+{% set version = "0.2.0.dev0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://test-files.pythonhosted.org/packages/8a/fc/32b4aa2c9fa8e38417c6e0505600958edef1334c2485c388a46f7ce596ae/pymongoarrow-0.2.0.dev0.tar.gz
+  sha256: 21be8c0d01ce0b740215c6fe279423d41a4fc839c625e08ef7866d7b11aa4d1a
+
+build:
+  number: 0
+  script:
+    - export MONGO_NO_COPY_LIBARROW=1            # [unix]
+    - export MONGO_LIBARROW_DIR="${PREFIX}/lib"  # [unix]
+    - export MONGO_NO_COPY_LIBBSON=1             # [unix]
+    - export MONGO_LIBBSON_DIR="${PREFIX}"       # [unix]
+    - set MONGO_NO_COPY_LIBARROW=1               # [win]
+    - set "MONGO_LIBARROW_DIR=%LIBRARY_LIB%"     # [win]
+    - set MONGO_NO_COPY_LIBBSON=1                # [win]
+    - set "MONGO_LIBBSON_DIR=%LIBRARY_PREFIX%"   # [win]
+    - {{ PYTHON }} -m pip install . --no-deps -vvv
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython  >=0.29                         # [build_platform != target_platform]
+    - arrow-cpp  <6.1,>=6                    # [build_platform != target_platform]
+    - pyarrow  <6.1,>=6                      # [build_platform != target_platform]      
+    - libbson  >=1,<2                        # [build_platform != target_platform] 
+    - pkg-config                             # [build_platform != target_platform] 
+    - setuptools >=47              
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+   
+  host:
+    - python
+    - arrow-cpp <6.1,>=6
```

Comment:
Is this only compatible with a single arrow version? Otherwise if you omit the pin here, we would build the package automatically for the last three arrow releases.

Reply:
We only test against 6.x currently, I wouldn't feel comfortable opening it up otherwise.